src/main/java/address/prefs/PrefsManager.java

```diff
@@ -0,0 +1,94 @@
+package address.prefs;
+
+import address.events.EventManager;
+import address.events.MirrorLocationChangedEvent;
+import address.events.SaveLocationChangedEvent;
+
+import java.util.prefs.Preferences;
+import java.io.File;
+
+/**
+ * Manages saving/retrieving of preferences on the user level.
+ * Publicly accessible singleton class.
+ */
+public class PrefsManager {
+
+    public static final String PREF_NODE_NAME_SAVE_LOC = "save-location";
+    public static final String PREF_NODE_NAME_MIRROR_LOC = "mirror-location";
```

Comment:
Maybe we don't need to mention `NODE_NAME` as the base implementation might change. `SAVE_LOC_PREF_KEY` might be better? I assume this is some key-value pairing data.